src/Nethermind/Nethermind.Core/BlockHeader.cs

```diff
@@ -83,6 +83,10 @@ public class BlockHeader
         // ToDo we need to set this flag after reading block from db
         public bool IsPostMerge { get; set; }
 
+        // transient:
+        public UInt256 Fees { get; set; }
```

Comment:
Do we really need to add these fields here? It looks like that it should be in tracer

Reply:
I do agree, but then we need to move it from tracer to handler, do you have better idea to do that?

Reply:
Maybe we can do it better, foundation here: https://github.com/NethermindEth/nethermind/tree/feature/fees_in_getPayload-refactor